src/api/Models/Role.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace BDMS.Models;
+﻿using System.Text.Json.Serialization;
+
+namespace BDMS.Models;
 
 /// <summary>
 /// Represents a role entity in the database.
 /// </summary>
+[JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
Q: Ich habe in meinem aktuellen Branch den einen Dateonly Converter direkt im Programm.cs eingebunden. Machen wir das i.d.R lieber direkt im Modell oder ist das egal?

Reply:
Wenn man den Converter generell verwenden möchte, dann ist die Einbindung in `Program` richtig. Da ich nicht weiss, ob wir den `JsonStringEnumConverter` auch für andere `enum`s verwenden werden, habe ich ihn direkt auf der `Role` definiert. Diesen müsste man dann auch ins `Program` zügeln, sobald wir ihn generell einsetzen möchten.